Sources/Amplitude/AMPURLConnection.h

```diff
@@ -1,34 +0,0 @@
-//
-//  AMPURLConnection.h
```

Comment:
What about the file `AMPURLConnetion.m`?

Reply:
Thanks for reviewing! Good catch, I thought I have removed it and committed the removal (it doesn't show up in my Xcode). Updated.